lib/Doctrine/DBAL/Connection.php

```diff
@@ -96,6 +96,13 @@ class Connection implements DriverConnection
     protected $_conn;
 
     /**
+     * The connection name.
+     *
+     * @var string
+     */
+    protected $_name;
```

Comment:
why adding a name in the connection ? This does not make sense IMO. there is nothing requiring a connection name in Doctrine